CITATION.cff

```diff
@@ -0,0 +1,56 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: 'NetCDF-C '
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - name: NSF Unidata
+    website: 'https://unidata.ucar.edu'
+  - given-names: Russ
+    family-names: Rew
+    affiliation: UCAR/Unidata
+  - given-names: Dennis
+    family-names: Heimbigner
+    affiliation: UCAR/Unidata
+  - given-names: Edward
+    family-names: Hartnett
+    affiliation: 'UCAR/Unidata, NOAA'
+  - given-names: Ward
+    family-names: Fisher
```

Comment:
Glenn Davis also should be listed.

I suggest you take the list of names from the user guide in that order.

Reply:
OK never mind I see he is there. ;-)

Reply:
No problem; I said I was putting a pin in it so of course I then went and immediately took care of it so I could cross it off and focus on the other dozen pinned items XD